src/components/Map/YMap/YMap.ts

```diff
@@ -105,17 +104,27 @@ export class YMap {
             rightTop = [Math.max(rightTop[0], point[0]), Math.max(rightTop[1], point[1])];
         });
 
-        const newMapParams = window.ymaps.util.bounds.getCenterAndZoom(
-            [leftBottom, rightTop],
-            [this.mapRef?.clientWidth, this.mapRef?.clientHeight],
-            undefined,
-            {margin: DEFAULT_MAP_CONTROL_BUTTON_HEIGHT},
-        );
+        let newMapParams = {
+            zoom: zoom || 0,
+            center: [],
+        };
+
+        if (zoom) {
+            // compute only the center
+            newMapParams.center = window.ymaps.util.bounds.getCenter([leftBottom, rightTop]);
```

Comment:
can we get the situation when window.ymaps or window.ymaps.util or window.ymaps.util.bounds is undefined?